src/test/kotlin/org/malachite/estella/queue/TaskResultTest.kt

```diff
@@ -80,13 +80,25 @@ class TaskResultTest : BaseIntegration() {
     @Order(1)
     fun `task result consuming`() {
         expectThat(taskStage.tasksResult.size).isEqualTo(0)
+        val code = SerialClob("xd".toCharArray())
```

Comment:
Może to zabrzmi głupio, ale zrobiłbym 
`val xd1 = "xd"`
`val xd2 = "xdd"`
Czułbym się bezpieczniej, kiedy stałe pilnują, żeby beki nigdy nie było ani za mało, ani za dużo